pages/timer.js

```diff
@@ -58,17 +34,15 @@ export default function TimerPage() {
         timerChain={timerChain}
         timerPointer={timerPointer}
       />
-      <FormHeadline />
-      <EditTimerChainForm
-        timerChain={timerChain}
-        setTimerChain={setTimerChain}
-        setTimerPointer={setTimerPointer}
-        handleSubmit={handleSubmit}
-        countdownKey={countdownKey}
-        setCountdownKey={setCountdownKey}
-      />
+      <EditWrapperWrapper>
+        <EditWrapper>
+          <Link href="/edit">
+            <Image alt="edit" src={edit} width="64" height="64" />
+          </Link>
+        </EditWrapper>
+      </EditWrapperWrapper>
```

Comment:
You call this in the timer and index page. You could think about creating a component that you can call (like the backgroundimage) on both pages instead of writing the code twice. But it's not super important.

Reply:
Good point I'll do it later :)